### PR TITLE
data: TusharKesarwani

### PR DESCRIPTION
**Pull request by TusharKesarwani**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4390"></a>

### CODE REVIEW

File: data/TusharKesarwani.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Tushar Kesarwani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Problem Setter - Coding Intern at iMocha || Front-End Developer || Competitive Programmer || 4 star at CodeChef || Pupil at Codeforces",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/TusharKesarwani",
+      "icon": "FaGitHub"
```
Comment (rupali-codes) [inline data/TusharKesarwani.json:10]:
```suggestion
      "icon": "FaGithub"
```

**Review by rupali-codes — APPROVED** [review]
looks great to me

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**